common/persistence/visibility/store/elasticsearch/processor.go

```diff
@@ -220,25 +216,25 @@ func (p *processorImpl) bulkBeforeAction(_ int64, requests []elastic.BulkableReq
 func (p *processorImpl) bulkAfterAction(_ int64, requests []elastic.BulkableRequest, response *elastic.BulkResponse, err error) {
 	if err != nil {
 		const logFirstNRequests = 5
-		httpStatus := client.HttpStatus(err)
-		isRetryable := client.IsRetryableStatus(httpStatus)
+		var httpStatus int
+		if err, isElasticErr := err.(*elastic.Error); isElasticErr {
```

Comment:
nit: Use `errors.As`. It's safer because it works even when the errors are wrapped